src/index.html

```diff
@@ -6,7 +6,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day--wed calendar--month-length--29">
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+      <div class="calendar__cell"></div>
+    </div>
   </body>
+
 </html>
```

Comment:
```suggestion
  </body>
</html>
```
Don't leave empty lines between parent and child elements